### PR TITLE
add: Product Listing Page

### PR DESCRIPTION
**Pull request by shilpe26**
Made Product Listing Page with features like:-
🔹Sidebar Filters-> Category, Sort by, Rating
🔹Cards
Screenshot (250)
Screenshot (251)

### CODE REVIEW

File: components/bookstore.html [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+        <section class="filter-drawer">
+            <div class="filter-header">
+                <h3>Filters</h3>
+                <button class="btn">Clear</button>
+            </div>
+            <span class="hr_line-book"></span>
+            <div class="filter-category filter">
+                <h4>Category</h4>
+                <label for="fiction">
+                    <input type="checkbox" id="fiction" value="1">Fiction
```
Comment (shrey27) [inline components/bookstore.html:62]: give some space between input and labels

**Review by shrey27 — APPROVED** [review]

File: components/bookstore.html [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+                    <input type="radio" id="rating-1" value="1" name="rating">1 Stars & above
+                </label>
+            </div>
+        </section>
+        <!-- Card section starts -->
+        <section class="cards_for-book">
+
+            <div class="card-vertical flex-items">
+                <div class="image-container badge-container">
+                    <img class="img-responsive card-img" src="../assest/fiction-book2.jpg" alt="">
```
Comment (anuj-kumary) [inline components/bookstore.html]: Always give name to **alt**

File: components/bookstore.html (hunk 1 above)
Comment (shilpe26) [reply components/bookstore.html:62]: okay, done

File: components/bookstore.html (hunk 2 above)
Comment (shilpe26) [reply components/bookstore.html]: Done

Comment (anuj-kumary) [reply components/bookstore.html]: Screenshot from 2022-02-13 18-03-38
Why this extra search icon

Comment (shilpe26) [reply components/bookstore.html]: That extra one is for js functionality I was trying to hide it but was unable to do so.
Will try again.

Comment (anuj-kumary) [reply components/bookstore.html]: I think you can give display none and whenever you want this give display block by using JS or media query

**Review by anuj-kumary — APPROVED** [review]

File: components/bookstore.html (hunk 2 above)
Comment (shilpe26) [reply components/bookstore.html]: Yes, I do but I try it again👍🏻

Comment (shilpe26) [reply components/bookstore.html]: @anuj-kumary  Done👍🏻
Thank you for highliting

**Pull request merged**